src/utils.ts

```diff
@@ -225,7 +220,7 @@ export function getRemoteEndpointSettings(config: Partial<IUserConfig> = {}): ID
         }
     }
 
-    if (userDataDir === true || (typeof userDataDir === "undefined" && !browserPath)) {
+    if (userDataDir === true || (typeof userDataDir === "undefined" && browserPathSet)) {

```

Comment:
Correct me if I am wrong, but I think we either want to flip the boolean here or in browserPathSet above.

browserPath would return false if the user used default and would return true if the user had an alternate path
browserPathSet would also return false if the browserFlavor was default and return true if the user set it to anything else but default

It looks like these booleans return true/false for default/not default, so we shouldn't remove the "!" from the if statement.

Reply:
good catch!